services/backend/src/routes/connectivity.js

```diff
@@ -114,6 +114,43 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/timeline TimelineByCategory
+   * @apiName TimelineByCategory
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of protected and protected connected of all years for a given area
+   *
+   * @apiParam (Query params) {String|Number} areaType area type
+   * @apiParam (Query params) {String} areaId area id
+   * @apiParam (Query params) {String} category category of connectivity index
+   *
+   * @apiSuccess {Object[]} result
+   * @apiSuccess {String} result.id PA connectivity category
+   * @apiSuccess {Object[]} result.data timeline data
```

Comment:
You need to describe also the elements of data. The only case we don't describe them is when it's a GeoJSON

Reply:
Elements of data and result type changed.